common/jack/jack.h

```diff
@@ -1420,8 +1420,9 @@ jack_time_t jack_get_time(void) JACK_OPTIONAL_WEAK_EXPORT;
  *
  * @param msg error message text (no newline at end).
  */
-extern void (*jack_error_callback)(const char *msg) JACK_OPTIONAL_WEAK_EXPORT;
-
+#ifndef _WIN32
```

Comment:
Firstly, this should be _MSC_VER, not _WIN32, but are you sure you can't make it work?

Reply:
I'll try, forgot about this change, so it could be I did it trying to solve something totally unrelated and forgot to revert.